deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -3446,41 +3447,41 @@ NAN_METHOD(WebGLRenderingContext::Finish) {
 }
 
 NAN_METHOD(WebGLRenderingContext::VertexAttrib1f) {
-  GLuint indx = info[0]->Int32Value();
-  GLfloat x = info[1]->NumberValue();
+  GLuint indx = TO_UINT32(info[0]);
+  GLfloat x = TO_FLOAT(info[1]);
 
   glVertexAttrib1f(indx, x);
 }
 
 NAN_METHOD(WebGLRenderingContext::VertexAttrib2f) {
-  GLuint indx = info[0]->Int32Value();
-  float x = (float)info[1]->NumberValue();
-  float y = (float)info[2]->NumberValue();
+  GLuint indx = TO_UINT32(info[0]);
```

Comment:
Should this be `TO_INT32` or was this intentional to match `GLuint` type?

Reply:
I think the spec says `GLuint` for all of these: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/vertexAttrib#Parameters